packages/clay-layout/src/Container.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Adds `.container-fluid` class to create a fluid container that
+	 * doesn't expand beyond a set width
+	 */
+	fluid?: boolean;
+
+	/**
+	 * Adds the `.container-form-${formSize}` class to properly space
+	 * between application controls and the form. This class only modifies
+	 * the padding on the container.
+	 */
+	formSize?: 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
This is actually an interesting take... we treated this as `className` but it might make sense as an API...

@eduardoallegrini, thoughts?

Reply:
I went through our documentation and tried to expose API that we explicitly talk about in the css or in our layouts definition. This is why I added it.

Reply:
I think it can be confusing with form sizes, if we speak about 'padding or margin' this probably should be 'spacers'. We have 8 unit sizes defined:
https://github.com/liferay/clay/blob/0273c22d1f6a9dfbd22e39f9490d7a9da05e53b7/packages/clay-css/src/scss/atlas/variables/_globals.scss#L182-L211
It generates all famous helpers like: `m-1`, `pt-5`, and our won utilities helpers: `c-m-5`, `c-pt-5`, etc.

Info about forms size:

For forms we have two groups sizes, and it can be consider a container: the default one `form-group` and `form-group form-group-sm`

for individual forms (form-controls) we have three sizes:
![image](https://user-images.githubusercontent.com/7963804/81699352-814beb00-9467-11ea-886b-cdcc808cea50.png)

Reply:
I would remove this one, just on the basis that it's easier to add them later than to remove them.

We can implement for now using `className`, and can bring it as an API once the pattern is more clear?